segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ReadIndexCache.java

```diff
@@ -362,11 +373,14 @@ private void evictChunks(SegmentReadIndex segmentReadIndex, long toRemoveCount)
      * The underlying data structure uses ConcurrentSkipListMap.
      * The generation is used to cleanup unused entries.
      */
-    @Builder
-    @Data
+    @RequiredArgsConstructor
```

Comment:
`@Data` is the same as `@RequiredArgsConstructor` + `@Getter`. You could have left that and save the extra Getters below.

Reply:
ok. added back @Data